aldryn_newsblog/managers.py

```diff
@@ -16,13 +18,16 @@ def get_query_set(self):
 
     def get_months(self, namespace):
         """
-        Get months with posts count for given namespace string.
+        Get months and years with posts count for given namespace string.
 
-        This means how much posts there are in each month. Results are ordered
-        by date.
+        This means how much posts there are in each month.
+        Returns list of dictionaries of the following format:
+        [{'date': date(YEAR, MONTH, ARBITRARY_DAY), 'num_entries': NUM_ENTRIES}, ...]
+        ordered by date.
         """
-        # done in a naive way as Django is having tough time while aggregating
-        # on date fields
+
+        # This is done in a naive way as Django is having tough time while
+        # aggregating on date fields
```

Comment:
Did not check the changelog, TBH. Can we tune this up a bit later, as it will change the return value signature (and break some templates) for sure?